Userland/Libraries/LibGfx/ImageFormats/JBIG2Loader.cpp

```diff
@@ -701,6 +701,21 @@ static ErrorOr<void> validate_segment_combination_operator_consistency(JBIG2Load
     return {};
 }
 
+// 7.4.10 End of stripe segment syntax
+struct [[gnu::packed]] EndOfStripeSegment {
```

Comment:
Is this `[[gnu::packed]]` actually needed?
(Note; BigEndian is already packed for some reason)

Reply:
This one isn't, but the other two segment data structs in this file that I `AssertSize` on and pointer-cast to do need it, due to having an `u8 flags` in the middle (or at the end). And it's maybe nice to be consistent about this in this file. (No very strong opinion here though. But that's why it's there.)